src/Core/ServiceWrapper/Main.cs

```diff
@@ -276,6 +277,11 @@ protected override void OnStart(string[] args)
                 startarguments += " " + _descriptor.Arguments;
             }
 
+            // Converting newlines, line returns, tabs into a single 
+            // space. This allows users to provide multi-line arguments
+            // in the xml for readability.
+            startarguments = Regex.Replace(startarguments, @"\s*[\n\r\t]+\s*", " ");

```

Comment:
```suggestion
            startarguments = Regex.Replace(startarguments, @"\s*[\n\r]+\s*", " ");
```

Tabs aren't special.

Reply:
Updated the pull request.